packages/components/src/components/select-native/select-native.jsx

```diff
@@ -102,7 +103,17 @@ const SelectNative = ({
                 {error && <Field message={error} type='error' />}
             </div>
         </div>
-        {!error && hint && <p className='dc-select-native__hint'>{hint}</p>}
+        {!error && hint && (
+            <Text
+                as='p'
+                color='less-prominent'
+                size='xxs'
+                styles={{ lineHeight: '1.8' }}
+                className='dc-select-native__hint'
```

Comment:
Question: is it possible to add a new lineHeight as maybe `--text-lh-xl` variable and normalize this as any other `line_height` ?

Reply:
For special cases we can pass the props like this: `styles={{ lineHeight: 'xl' }}`